PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift

```diff
@@ -0,0 +1,69 @@
+//
+// This source file is part of the CS342 2023 PAWS Team Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+let backgroundGradient =  LinearGradient(
+    colors: [.red, .pink, .orange, .yellow],
+        startPoint: .leading,
+        endPoint: .trailing
+    )
+
+public struct LandingScreen: View {
+//    @AppStorage(StorageKeys.accountCreated) var completedAccountSetup = false
+    @Binding private var launchStatus: Bool
+    public var body: some View {
+        VStack {
+            backgroundGradient
+            .mask(
+                VStack {
+                    Image(systemName: "pawprint.circle.fill")
+                        .resizable()
+                        .scaledToFill()
+                        .frame(width: 200, height: 200)
+                        .foregroundColor(.red)
+                    Text("PAWS")
+                        .font(.custom("GillSans-Bold", fixedSize: 30))
+                    Text("The Pediatric Apple Watch Study")
+                        .font(.custom("GillSans", fixedSize: 15))
+                        .offset(y: 50)
+                    
+                   
+                    
+                }
+            )
+       
+            Button(action: {
+                launchStatus = true
+            }) {
+                
+                Text("Tap to get started")
+                    .fontWeight(.bold)
+                    .padding()
+                    .foregroundColor(Color.red)
+                    .border(backgroundGradient, width: 3)
+                    .cornerRadius(5)
+                    .offset(y: -30)
+            }
+
```

Comment:
I recommend the following formatting for a better readability of buttons and SwiftUI views in general:
```suggestion
            Button(
                action: {
                    launchStatus = true
                },
                label: {
                    Text("Tap to get started")
                        .fontWeight(.bold)
                        .padding()
                        .foregroundColor(Color.red)
                        .border(backgroundGradient, width: 3)
                        .cornerRadius(5)
                        .offset(y: -30)
                }
            )
```